services/app-web/src/pages/LinkYourRates/LinkRateSelect.tsx

```diff
@@ -106,15 +115,23 @@ export const LinkRateSelect = ({
         }
     }
 
+    //Need this to make the label searchable since it's buried in a react element
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const filterOptions = ({ label }: any, input: string) =>
```

Comment:
what does this do?

Reply:
You can use `filterOption` to customize how the search filtration logic works for the dropdown menu. The dropdown menu search works on label and value _strings _ but the label is currently a react element. So I leveraged the filterOptions prop to dig into the react element so it can search by the rate name. I had to include `.toLowerCase()` to keep it case-insensitive as well.